lib/importproject.cpp

```diff
@@ -1318,7 +1316,7 @@ bool ImportProject::importCppcheckGuiProject(std::istream &istr, Settings &setti
         else if (strcmp(name, CppcheckXml::PlatformElementName) == 0)
             guiProject.platform = empty_if_null(node->GetText());
         else if (strcmp(name, CppcheckXml::AnalyzeAllVsConfigsElementName) == 0)
-            guiProject.analyzeAllVsConfigs = empty_if_null(node->GetText());
+            temp.analyzeAllVsConfigs = std::string(empty_if_null(node->GetText())) != "false";
```

Comment:
ideally I guess we should abort with an error if it says something strange like "xyz".

Reply:
I wanted to keep the old behavior intact for now. The GUI part is currently lacking testing but adding it should become unnecessary when CLI and GUI behavior is aligned with #7293.

Reply:
Actually we are missing proper errorhandling for the GUI project reading and thus also lack the tests. We only make sure that it doesn't crash on invalid inputs.

Reply:
ok I don't think you need to fix it in this PR